### PR TITLE
improve the logging format

### PR DESCRIPTION
**Pull request by runzhliu**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/discovery/token/token.go [hunk 1]
```diff
@@ -208,7 +208,7 @@ func getClusterInfo(client clientset.Interface, kubeconfig *clientcmdapi.Config,
 	}
 
 	klog.V(1).Infof("[discovery] Waiting for the cluster-info ConfigMap to receive a JWS signature"+
-		"for token ID %q", token.ID)
+		" for token ID %q", token.ID)
```
Comment (googs1025) [inline cmd/kubeadm/app/discovery/token/token.go:211]: Do we use `fmt.sprintf` to optimize this hard-coded style?
```go
msg := fmt.Sprintf("[discovery] Waiting for the cluster-info ConfigMap to receive a JWS signature for token ID %q", token.ID)
klog.V(1).Infof(msg)
```

Comment (runzhliu) [reply cmd/kubeadm/app/discovery/token/token.go:211]: agree

**Comment by carlory** [comment]
Do not contain merge commits, please use `git rebase` to pull the latest changes from the master branch.

**Comment by runzhliu** [comment]
> Do not contain merge commits, please use `git rebase` to pull the latest changes from the master branch.

ok

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**